pkg/controller/volume/pvcprotection/pvc_protection_controller.go

```diff
@@ -256,6 +259,29 @@ func (c *Controller) pvcAddedUpdated(obj interface{}) {
 	}
 }
 
+func (c *Controller) pvcDeleted(obj interface{}) {
+	tombstone, ok := obj.(cache.DeletedFinalStateUnknown)
```

Comment:
this isn't correct... this should first check if obj is an instance of v1.PVC. if not, *then* check if it's a tombstone and can extract the item